src/desktop/apps/artwork/client/index.coffee

```diff
@@ -1,7 +1,7 @@
 { extend, map, compact } = require 'underscore'
 { AUCTION, CLIENT } = require('sharify').data
 { setCookie } = require '../../../components/recently_viewed_artworks/index.coffee'
-{ recordArtworkView } = require 'lib/components/record_artwork_view'
+{ recordArtworkView } = require '../../../../lib/components/record_artwork_view'
```

Comment:
I couldn't run the tests locally unless this used a relative path (seems like it's been passing on CI though, so could just be me?)

Reply:
Maybe some compiler caching going on? Unsure off-hand without being at the computer :|

Reply:
There's a relevant discussion here when it was added: https://github.com/artsy/force/pull/2355/files#r181465475 seems okay to revert to relevant paths